freetextresponse/templates/freetextresponse_view.html

```diff
@@ -30,16 +30,24 @@ <h2 class="problem-header">{{ display_name }}</h2>
     </div>
     <div class="capa_alert user_alert">{{ user_alert }}</div>
     {% if display_other_responses %}
-        <div class="responses-box">
-            <button class="hide-button">
-                <span class="hide">Hide</span>
-                <span class="show">Show</span>
-                <span class="sr">peer responses</span>
-            </button>
-            <p class="responses-title">Submissions by others</p>
-            <ul class="response-list">
-                <li class="no-response">No responses to show at this time</li>
-            </ul>
-        </div>
+        {% if student_answer %}
+            <div class="responses-box">
```

Comment:
I think this got over-engineered.
What you had here (class name in a variable) previously was fine.
In fact, it's what we want, to specify visibility with a variable.

I was just pointing out that it should be the template itself which assigns the variable,
not the backend.